repose-aggregator/components/filters/attribute-mapping-policy-validation-filter/src/main/scala/org/openrepose/filters/attributemapping/AttributeMappingPolicyValidationFilter.scala

```diff
@@ -0,0 +1,146 @@
+/*
+ * _=_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_=
+ * Repose
+ * _-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-
+ * Copyright (C) 2010 - 2015 Rackspace US, Inc.
+ * _-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * =_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_=_
+ */
+package org.openrepose.filters.attributemapping
+
+import java.io.{ByteArrayInputStream, ByteArrayOutputStream, InputStream}
+import javax.inject.Named
+import javax.servlet._
+import javax.servlet.http.HttpServletResponse.{SC_BAD_REQUEST, SC_UNSUPPORTED_MEDIA_TYPE}
+import javax.servlet.http.{HttpServletRequest, HttpServletResponse}
+import javax.xml.transform.stream.{StreamResult, StreamSource}
+import javax.xml.transform.{Source, TransformerException}
+
+import com.fasterxml.jackson.core.{JsonParseException, JsonProcessingException}
+import com.rackspace.identity.components.{AttributeMapper, XSDEngine}
+import com.typesafe.scalalogging.slf4j.LazyLogging
+import net.sf.saxon.s9api.{SaxonApiException, XdmDestination}
+import org.openrepose.commons.utils.io.stream.ServletInputStreamWrapper
+import org.openrepose.commons.utils.servlet.http.HttpServletRequestWrapper
+
+import scala.util.{Failure, Success, Try}
+
+@Named
+class AttributeMappingPolicyValidationFilter extends Filter with LazyLogging {
+
+  import AttributeMappingPolicyValidationFilter._
+
+  // Magic
+  private val transformerFactory = new net.sf.saxon.TransformerFactoryImpl
```

Comment:
I'm very confused as to why you need a transformer in the first place. Will look this over tomorrow.